Makefile

```diff
@@ -8,14 +8,13 @@ install:
 
 .PHONY: test
 test:
-	go test -race $(PACKAGES) -count 2
+	$(TEST) ./...
 
 .PHONY: ci
-ci: SHELL := /bin/bash
 ci:
 ifdef COVER
-	$(TEST) -cover -coverprofile=coverage.txt $(PACKAGES)
-	# bash <(curl -s https://codecov.io/bash)
+	$(TEST) -cover -coverprofile=coverage.txt ./...
+	bash <(curl -s https://codecov.io/bash)
```

Comment:
Should this be `$(SHELL) <(curl -s https://codecov.io/bash)` ?

Reply:
No, the `SHELL` variable is special: https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html